examples/main.cpp

```diff
@@ -700,6 +700,14 @@ int main(int argc, char **argv)
     CALIPER(CALI_MARK_END("Cycle");)
     MPI_CALL(MPI_Barrier(MPI_COMM_WORLD));
   }
+
+  delete [] workflow;
+
+  for ( int mat_idx = 0; mat_idx < num_mats; ++mat_idx){
+    delete eoses[mat_idx];
```

Comment:
I'll accept this but add a TODO to use smart pointers

Reply:
Fixed